src/benchmark.cc

```diff
@@ -571,25 +637,31 @@ void ParseCommandLineFlags(int* argc, char** argv) {
   BenchmarkReporter::Context::executable_name = argv[0];
   for (int i = 1; i < *argc; ++i) {
     if (ParseBoolFlag(argv[i], "benchmark_list_tests",
-                      &FLAGS_benchmark_list_tests) ||
-        ParseStringFlag(argv[i], "benchmark_filter", &FLAGS_benchmark_filter) ||
+                      &FLAG(benchmark_list_tests)) ||
+        ParseStringFlag(argv[i], "benchmark_filter", &FLAG(benchmark_filter)) ||
         ParseDoubleFlag(argv[i], "benchmark_min_time",
-                        &FLAGS_benchmark_min_time) ||
+                        &FLAG(benchmark_min_time)) ||
         ParseInt32Flag(argv[i], "benchmark_repetitions",
-                       &FLAGS_benchmark_repetitions) ||
+                       &FLAG(benchmark_repetitions)) ||
         ParseBoolFlag(argv[i], "benchmark_report_aggregates_only",
-                      &FLAGS_benchmark_report_aggregates_only) ||
-        ParseStringFlag(argv[i], "benchmark_format", &FLAGS_benchmark_format) ||
-        ParseStringFlag(argv[i], "benchmark_out", &FLAGS_benchmark_out) ||
+                      &FLAG(benchmark_report_aggregates_only)) ||
+        ParseStringFlag(argv[i], "benchmark_format", &FLAG(benchmark_format)) ||
+        ParseStringFlag(argv[i], "benchmark_out", &FLAG(benchmark_out)) ||
         ParseStringFlag(argv[i], "benchmark_out_format",
-                        &FLAGS_benchmark_out_format) ||
-        ParseStringFlag(argv[i], "benchmark_color", &FLAGS_benchmark_color) ||
+                        &FLAG(benchmark_out_format)) ||
+        ParseBoolFlag(argv[i], "benchmark_console_id",
+                      &FLAG(benchmark_console_id)) ||
+        ParseBoolFlag(argv[i], "benchmark_console_family",
+                      &FLAG(benchmark_console_family)) ||
+        ParseBoolFlag(argv[i], "benchmark_console_base_name",
+                      &FLAG(benchmark_console_base_name)) ||
+        ParseStringFlag(argv[i], "benchmark_color", &FLAG(benchmark_color)) ||
         // "color_print" is the deprecated name for "benchmark_color".
         // TODO: Remove this.
-        ParseStringFlag(argv[i], "color_print", &FLAGS_benchmark_color) ||
+        ParseStringFlag(argv[i], "color_print", &FLAG(benchmark_color)) ||
```

Comment:
there's no need to introduce this `FLAG(..)` macro. It doesn't add anything useful.

Reply:
I didn't introduce the FLAG macro; this was present in the source already. It was defined as "the way you are supposed to use it to refer to flags". I simply noticed that it was defined as the way to do things and we didn't use it anywhere. I will revert the changes, but does that mean the `FLAG` macro present [here](https://github.com/google/benchmark/blob/master/src/commandlineflags.h#L8) should not be around...?